react/bingo/src/bingo/BingoApp.css

```diff
@@ -0,0 +1,74 @@
+.bingo {
```

Comment:
Move css files to styles folder

Reply:
Moved the css files to the styles folder in the new project. Hence resolving the discussion.